test/plots/software-versions.js

```diff
@@ -0,0 +1,31 @@
+import * as Plot from "@observablehq/plot";
+import * as d3 from "d3";
+
+export default async function() {
+  const data = await d3.csv("data/software-versions.csv");
+  const options = Plot.stackX(
+    Plot.groupZ(
+      {
+        x: "proportion", // uses sumOrCount in the patched version
+        text: "first"
+      },
+      {
+        x: "key", // this is only strings
```

Comment:
This isn’t strictly needed here because it doesn’t affect the composition of the groups (which here for groupZ is determined only by the _fill_ channel), i.e. removing this key doesn’t affect the chart (but with it removed, we’re no longer testing the feature added by this PR). It’d be nice to have a more explicit test.